python/apps/backend/backend/config.py

```diff
@@ -27,8 +27,8 @@
 AZURE_CLIENT_ID = os.environ.get("AZURE_CLIENT_ID", None)
 AZURE_CLIENT_SECRET = os.environ.get("AZURE_CLIENT_SECRET", None)
 AZURE_TENANT_ID = os.environ.get("AZURE_TENANT_ID", None)
-AZURE_AUTHORITY = f"https://login.microsoftonline.com/common"
-AZURE_DISCOVERY_URL = "https://login.microsoftonline.com/common/v2.0/.well-known/openid-configuration"
+AZURE_AUTHORITY = os.environ.get("AZURE_AUTHORITY", f"https://login.microsoftonline.com/{AZURE_TENANT_ID}")
```

Comment:
Interesting. /common still works fine for me. I didn't go through the issue you mentioned in stack overflow, but I think I might have some different Azure configuration.. (not an Azure expert :p).